integration-test/test-pages/broker-protection/actions/fill-form.json

```diff
@@ -12,6 +12,10 @@
         {
           "type": "lastName",
           "selector": "#user_last_name"
+        },
+        {
+          "type": "phone",
```

Comment:
I used a `$` prefix + suffix in a previous PR to differentiate these generated fields from data lookups - we should do the same here, just make it something like `"type": "$generated_phone_number$" or similar. Otherwise it looks like 'phone' is coming from the data object

Reply:
Ah, I hadn't originally understood why you went with that naming convention, now it makes total sense. Updated.